File: server/app/models/Room.java
package models;

import javax.persistence.Entity;

import play.db.jpa.Model;

@Entity
public class Room extends Model {
	
	public String room_number;
	
	public String building;
	
	public String image;
	
	public String admin;
	
	public String day_start;
	
	public String day_end;
	
	public int capacity;
	
	
	
}

File: app/model/Room.js
Ext.define('RF.model.Room', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'id',           type: 'int',   useNull: true},
        {name: 'room_number',  type: 'string'},
        {name: 'building',     type: 'string'},
        {name: 'image',        type: 'string'},
        {name: 'admin',        type: 'string'},
        {name: 'day_start',    type: 'string'},
        {name: 'day_end',      type: 'string'},
        {name: 'capacity',     type: 'int'},
        'events'
    ],
    validations: [
        {type: 'presence',  field: 'first_name'},
        {type: 'presence',  field: 'last_name'},
        {type: 'presence',  field: 'device_id'},
        {type: 'presence',  field: 'phone_num'}
    ],
    proxy: {
        type: 'ajax',
        url : 'app/data/rooms.json'
    }
});